source/agora/node/Validator.d

```diff
@@ -137,9 +137,11 @@ public class Validator : FullNode, API
             assert(0);
         }
 
+        static const quorum_params = QuorumParams.init;
```

Comment:
Why the `static const` ?

Reply:
Temporary, it allowed me to split it into two commits.

It could have been just `const` but I didn't want to waste stack space. I guess it's pointless since it's removed in the next commit.